test/test_integration_credentials_netrc.py

```diff
@@ -15,20 +15,19 @@
 from conjur.data_object import CredentialsData
 from conjur.logic.credential_provider import FileCredentialsProvider
 from conjur.logic.credential_provider.credential_store_factory import CredentialStoreFactory
-from conjur.logic.credential_provider.CredentialProviderTypes import CredentialProviderTypes
 from test.util.test_infrastructure import integration_test
 from test.util.test_runners.integration_test_case import IntegrationTestCaseBase
 from test.util import test_helpers as utils
 
 from conjur.constants import DEFAULT_NETRC_FILE, DEFAULT_CONFIG_FILE, DEFAULT_CERTIFICATE_FILE
 
+
 @patch('conjur.wrapper.keystore_adapter.KeystoreAdapter.is_keyring_accessible', return_value=False)
```

Comment:
Please add comment above what this is doing because no one will know what this really means without really understanding the code

Something like:
```
"""
Some tests require that the Keyring on the system's environment is not accessible. This is required to validate that the CLI operates as expected under such conditions.
"""
```